sycl/test/on-device/basic_tests/specialization_constants/kernel_handler.cpp

```diff
@@ -0,0 +1,31 @@
+// RUN: %clangxx -fsycl  -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %t.out
+
+#include <CL/sycl.hpp>
+
+int main() {
+  sycl::queue q;
+
+  q.submit([&](sycl::handler &cgh) {
+    cgh.single_task<class Kernel1>([=](sycl::kernel_handler kh) {});
+  });
+
```

Comment:
Please, add all possible combinations of parallel_for arguments.

Reply:
Added